core/storage/suggestion/gae_models.py

```diff
@@ -313,6 +313,45 @@ def get_translation_suggestions_in_review_with_exp_id(
             cls.target_id == exp_id
         )).fetch(feconf.DEFAULT_QUERY_LIMIT)
 
+    @classmethod
+    def get_multiple_translation_suggestions_in_review(
+            cls, suggestion_ids: List[str]
+    ) -> List[Optional['GeneralSuggestionModel']]:
+        """Returns translation suggestions which are in review from multiple
+        thread ids.
```

Comment:
What is meant by "from multiple thread ids"?

Reply:
It should be suggestion IDs. Updated the comment.